### ISSUE TITLE
Add PDF export of individual notes

### ISSUE THREAD

**Issue by jamesrascal** [twostairs/paperwork#71]
Add the ability to create a PDF from an individual note.

**Comment by pwmvx1289** [comment]
There is a HTML to PDF converter in PHP.
https://github.com/dompdf/dompdf

**Comment by devilx** [comment]
Seems to look good @pwmvx1289. Every tried it yourself?

**Comment by Baldrs** [comment]
It is bullshit, mPDF is better and easier to use.

**Comment by pwmvx1289** [comment]
@devilx No, I just found it on the web, it seems to have rather comprehensive functions and simple usage.

**Comment by mrusme** [comment]
v2 now provides a way of exporting notes as PDFs simply by using the "Print" function, since every major OS provides [printing to PDF](https://www.digitaltrends.com/computing/print-pdf-windows/) these days.

Wondering if it's still relevant to have a dedicated PDF export and if so, why?
